### PR TITLE
Make Nettle need botanist gloves to be safely picked up

### PR DESCRIPTION
**Pull request by UKNOWH**
## About the PR 
If someone piced up a nettle without the botanist's glove , he would have part of the Histamine transfered to his body
**Screenshots**

**Changelog**

:cl:
- change: Be more careful with nettles , always wear the botanist glove before you try to pick them up

### CODE REVIEW

**Comment by 0x6273** [comment]
You still have some WriteLines left over from debugging

**Comment by UKNOWH** [comment]
> You still have some WriteLines left over from debugging

Ah I must have forgotten

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -361,3 +361,5 @@
 - type: Tag
   id: Write
 
+- type: Tag
+  id: NettleProtection #for gloves and gloves only
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:365]: This should be generically named.

File: Content.Server/Weapon/Melee/MeleeWeaponSystem.cs [hunk 2]
```diff
@@ -43,8 +49,25 @@ public override void Initialize()
...
+                return;
+            if (!_inventorySystem.TryGetSlotEntity(args.User, "gloves", out EntityUid? protection)
+                || !component.RequiredProtection.IsValid((EntityUid) protection))
+            {
+                //this is probably ugly, but I can't figure out a cleaner way to do it
+                List<EntityUid> thisList = new List<EntityUid>();
+                thisList.Add(args.User);
+                MeleeHitEvent thisEvent = new MeleeHitEvent(thisList, args.User);
+                TryInjectChemical(uid ,component ,thisEvent);
+            }
```
Comment (metalgearsloth) [inline Content.Server/Weapon/Melee/MeleeWeaponSystem.cs:67]: Make the method take in a list instead of just allocating an event.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/tags.yml (hunk 1 above)
Comment (UKNOWH) [reply Resources/Prototypes/tags.yml:365]: what do you mean by "generically named" ?

Comment (ike709) [reply Resources/Prototypes/tags.yml:365]: Something more generic like PointyObjectProtection

Comment (UKNOWH) [reply Resources/Prototypes/tags.yml:365]: ah thx

File: Content.Server/Weapon/Melee/MeleeWeaponSystem.cs (hunk 2 above)
Comment (UKNOWH) [reply Content.Server/Weapon/Melee/MeleeWeaponSystem.cs]: done

**Comment by UKNOWH** [comment]
This is fucked

**Comment by Froozigiusz** [comment]
> This is fucked

You mean only current code or the idea itslef? More utilization of clothes like this is a neat idea

**Comment by UKNOWH** [comment]
> 



> > This is fucked
>  mean only current code or the idea itslef? More utilization of clothes like this is a neat idea

I fucked up my git

**Pull request closed without merging**